CS342Application.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I have seen that you have added the `WrappingHStack` dependency to the main project without using it in your views. If you want to use it in your view, feel free to add it as a dependency to your target in the `Package.swift` file in the `MemberViews` Swift Package.
You can remove the dependency from the main application in the Xcode project settings (you can find more information about removing a dependency in the "Delete a package dependency" here: https://developer.apple.com/documentation/xcode/adding-package-dependencies-to-your-app).

This will then also remove this file and the references in the project settings.